packages/cbioportal-clinical-timeline/src/TimelineStore.tsx

```diff
@@ -60,6 +60,25 @@ export class TimelineStore {
         return this.collapsedTracks.has(trackUid);
     }
 
+    @computed get sampleEvents() {
+        return this.allItems.filter(
+            event => event.event!.eventType === 'SPECIMEN'
+        );
+    }
+
+    @computed get sampleIds() {
+        const sampleIds: string[] = [];
+        this.sampleEvents.forEach((sample, i) => {
```

Comment:
super minor but i like to use reduce for this kind of operation so that you get direct assignment, or even just direct return.  Matters more in longer routines.  You don't need to fix this.
```
const sampleIds = this.sampleEvents.reduce()